etc/avocado/avocado.conf

```diff
@@ -85,3 +85,7 @@ skip_broken_plugin_notification = []
 # options or test types).
 # The keyword "@DEFAULT" will be replaced with all available unused loaders.
 loaders = ['file', '@DEFAULT']
+
+[plugins.result]
+# Support for ordering result plugins *only* at this version
```

Comment:
I guess you mean "only result plugins" and not "only at this version". Please rephrase to remove the ambiguity.